dev-docs/adding-data/infra/chef-repo/data_bags/data/bag.json

```diff
@@ -0,0 +1,3 @@
+{
+  "id": "bag"
```

Comment:
Please make this named something other than data/bag too. In a rare case, it could be a "bug" if we did something like `knife data bag create foo" and ended up with a bag named "data".

Make the bag named 'colors' and the item 'purple' or something with a k/v of something like `"description": "this is an example data bag item."`

Reply:
I guess the data bag names are restricted only `node|role|environment|client` other than this it would be fine.

But this example is quite confusing so make sense to update with better example thankyou.